### PR TITLE
Unpacks ROOT.war for cloud debugger

### LINKED ISSUE #113: P0 - Cloud Debugger not enabled by default
@meltsufin 
Looking at how the Dockerfile you appear to do the right thing and inherit from OpenJDK8, but somehow when you start tomcat, it doesn't include all the right JAVAOPTS.

### PR DESCRIPTION
**Pull request by ChengyuanZhao**
fixes #113

### CODE REVIEW

File: tomcat/src/main/docker/50-tomcat.bash [hunk 1]
```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+# Unpack a WAR app (if present) beforehand so that Stackdriver Debugger
+# can load it. This should be done before the JVM for Tomcat starts up.
+export ROOT_WAR=$CATALINA_BASE/webapps/ROOT.war
+export ROOT_DIR=$CATALINA_BASE/webapps/root
```
Comment (saturnism) [inline tomcat/src/main/docker/50-tomcat.bash]: `root` should be all cap like `ROOT` right? otherwise we may have multiple extracts

**Review by saturnism — CHANGES_REQUESTED** [review]

File: tomcat/src/main/docker/15-debug-env-tomcat.bash [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Adds bits of configuration necessary for the Cloud Debugger to work with the Tomcat image.
+# It needs to run before 20-debug-env.bash from the openjdk image.
+
+# Set CDBG_APP_WEB_INF_DIR, used by CDBG in format-env-appengine-vm.sh
+export CDBG_APP_WEB_INF_DIR="${CATALINA_BASE}/webapps/root/WEB-INF"
```
Comment (meltsufin) [inline tomcat/src/main/docker/15-debug-env-tomcat.bash]: `root` -> `ROOT`

**Review by meltsufin — CHANGES_REQUESTED** [review]
Also need to disable Tomcat's upacking of WARs

**Comment by meltsufin** [comment]
I think you need to set `unpackWARs=false`.

File: tomcat/src/main/docker/15-debug-env-tomcat.bash (hunk 2 above)
Comment (ChengyuanZhao) [reply tomcat/src/main/docker/15-debug-env-tomcat.bash]: Ah yea  wasn't aware that tomcat unpacked it at all. Fixed

**Review by meltsufin — APPROVED** [review]

**Pull request merged**